cli/recovery.c

```diff
@@ -544,7 +544,7 @@ static int fw_transfer(int argc, char **argv)
 
 	print_fw_image_info(cfg.img_filename, &finfo);
 
-	ret = ask_if_sure(!cfg.confirm);
+	ret = ask_if_sure(cfg.confirm);
```

Comment:
I'd suggest to use the variable name 'assume_yes' to be consistent with the name used in fw-update CLI, which would be more meaningful.

Reply:
Code updated according to your suggestion. Please review.